README.md

```diff
@@ -321,7 +321,7 @@ angular.module('myApp', [
   'ui.router',
   'ng-token-auth'
 ])
-  .config(function($stateProvider) {
+  .config(function($stateProvider, $auth) {
```

Comment:
`$auth` is a service and therefore isn't injectable in the config phase (you could inject `$authProvider` though).

Reply:
`$auth` is injected on line 340, which happens in the run phase, so there is no need to inject it higher up.

Reply:
Ok. Thanks for comment.